src/products/components/ProductVariantEndPreorderDialog/ProductVariantEndPreorderDialog.tsx

```diff
@@ -2,7 +2,22 @@ import { DialogContentText } from "@material-ui/core";
 import ActionDialog from "@saleor/components/ActionDialog";
 import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
 import React from "react";
-import { useIntl } from "react-intl";
+import { defineMessages, useIntl } from "react-intl";
+
+const messages = defineMessages({
+  dialogTitle: {
+    defaultMessage: "Ending Preorder",
```

Comment:
```suggestion
    defaultMessage: "Ending preorder",
```
Should it be capitalized?

Reply:
fixed